pkg/operation/botanist/secrets.go

```diff
@@ -216,12 +216,25 @@ func (b *Botanist) generateCertificateAuthorities(ctx context.Context) error {
 		return fmt.Errorf("secret %q not found", v1beta1constants.SecretNameCACluster)
 	}
 
+	caBundle := caBundleSecret.Data[secretsutils.DataKeyCertificateBundle]
+
+	if err := b.syncShootConfigMapToGarden(
+		ctx,
+		gardenerutils.ShootProjectConfigMapSuffixCACluster,
+		map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleCACluster},
+		nil,
+		map[string]string{secretsutils.DataKeyCertificateCA: string(caBundle)},
+	); err != nil {
+		return err
+	}
+
+	// TODO(petersutter): Remove this code and cleanup Secret in a future release. The caBundle is now being stored in a <shootname>.ca-cluster ConfigMap.
```

Comment:
```suggestion
	// TODO(petersutter): Remove this code and cleanup Secret after v1.96 has been released. The caBundle is now being stored in a <shootname>.ca-cluster ConfigMap.
```

Maybe you can make yourself a reminder for CW23 :)

Reply:
✅  fixed and I created myself a reminder :)